CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed issue that caused certain operations not showing up in operations_log
+   table and certain operations showing up in operations_log table before the
```

Comment:
hm bit long and hard to understand, suggestion:
Fixed issue that caused certain operations not showing up at all in operations_log while other marked as finished even before they actually finished